index.html

```diff
@@ -265,7 +265,19 @@ <h4>Computation steps</h4>
                     <li id="step2B.ii">For each IDREF:
                       <ol>
                         <li id="step2B.ii.a">Set the <code>current node</code> to the node referenced by the IDREF.</li>
-                        <li id="step2B.ii.b">Compute the text alternative of the <code>current node</code> beginning with step 2. Set the <code>result</code> to that text alternative.</li>
+                        <li id="step2B.ii.b">
+                            Compute the text alternative of the <code>current node</code> using the following steps: 
+                            <ul>
+                                <li>If the <code>current node</code> is <strong>not</strong> <a class="termref">hidden</a>, recursively skip to step 2.</li>
```

Comment:
```suggestion
                                <li>If the <code>current node</code> is <a href="https://html.spec.whatwg.org/multipage/rendering.html#being-rendered">being rendered</a>, recursively skip to step 2.</li>
```

Reply:
@jnurthen so we would treat visibility:hidden as before, and display:none would get the new innerText treatment?

If that's what we decide to do, then I think we might want to call that out explicitly. It feels like the HTML definition would require interpretation by people who understand how browsers work.

Reply:
yeah - although having suggested this I since realised that this doesn't account for aria-hidden which is rendered but should probably fit into the non rendered category that uses innerText.

Reply:
@cookiecrook does this solve your implementation problem? Is HTML's definition of "rendered" what you need? 

@aleventhal are you good with this if we add examples for visibility:hidden and display:none? Maybe also html hidden attribute, opacity:0 and offscreen. I think it's easier to understand this from examples than from the algorithm, plus they can be the start of test files.

Reply:
That sounds good in general. If we're going to be complete, there is also content-visibility:hidden, I suppose.
I'm assuming that everyone agrees opacity:0 and offscreen are treated as not hidden.

Reply:
yes.  opacity: 0 and off-screen positioning (via position, negative text indent, etc) should not be treated the same as hidden.